### PR TITLE
NO NEED TO LOOK 'TIL MONDAY: Feature/start spec

### PR DESCRIPTION
**Pull request by mszenher**
@yzhangmedidata @HonoreDB Starting spec for delayed_job_sqs.  

About half of the shared specs pass right now.  This is mostly due to dj_sqs not supporting scheduled jobs (`run_at`) or job prioritization right now.

### CODE REVIEW

File: delayed_job_sqs.gemspec [hunk 1]
```diff
@@ -7,8 +7,8 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.name          = "delayed_job_sqs"
   s.version       = Delayed::Backend::Sqs.version
-  s.authors       = ["Eric Hankinson"]
-  s.email         = ["eric.hankinson@gmail.com"]
+  s.authors       = ["Eric Hankinson", "Matthew Szenher"]
+  s.email         = ["eric.hankinson@gmail.com", "mszenher@mdsol.com"]
```
Comment (mszenher) [inline delayed_job_sqs.gemspec:11]: @yzhangmedidata:  Please make a PR to add yourself here too.

File: lib/delayed/backend/worker.rb [hunk 2]
```diff
@@ -39,6 +39,10 @@ class Railtie < Rails::Railtie
             Delayed::Worker.configure {}
           end
         end
+      elsif Object.const_defined?(:AWS) && AWS.respond_to?(:config) && !(AWS.config.access_key_id.nil? || AWS.config.secret_access_key.nil?)
```
Comment (HonoreDB) [inline lib/delayed/backend/worker.rb]: This could be compressed to `defined?(AWS.config) && AWS.config.access_key_id && AWS.config.secret_access_key`

**Comment by HonoreDB** [comment]
Looks good to me (one non-blocking comment). Since @mszenher's out today, might merge if @yzhangmedidata doesn't have anything more substantive.

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+)
+
+require File.join(LIB_DIR, 'delayed_job_sqs')
+
+Delayed::Worker.logger = Logger.new('/tmp/dj.log')
+
+# TODO:  Shouldn't DJ_SQS just set the queue name(s)?
+Delayed::Worker.queues = [QUEUE_NAME]
+
+RSpec.configure do |config|
```
Comment (yzhangmedidata) [inline spec/spec_helper.rb:47]: Might be useful to have a few more comments around spec configs

**Comment by yzhangmedidata** [comment]
Looks good, just another non blocking comment

**Comment by mszenher** [comment]
@HonoreDB @yzhangmedidata Comments addressed.  Ready for final review and merge.

**Pull request merged**